contracts/BSCValidatorSet.sol

```diff
@@ -73,6 +73,26 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
     Validator[] validatorSet;
   }
 
+  // BEP-99 Temporary Maintenance
+  uint256 public constant EXIT_MAINTENANCE_REWARD_RATIO_SCALE = 100;
+  uint256 public constant EXIT_MAINTENANCE_REWARD_RATIO = 20;
+  uint256 public constant SAFE_MIN_GAS = 2300;
+
+  uint256 public MaxNumOfMaintaining;
+  uint256 public MaxMaintainingTime;
+
+  struct MaintainInfo {
+    // TODO optimize struct
+    bool isMaintaining;
+    uint256 maintainStartAt;
+    uint256 maintainingIndex;   // the index of the element in `maintainingValidatorSet`.
+    uint256 exitMaintenanceReward;
+  }
+  // key is the `consensusAddress` of `Validator`,
+  mapping(address => MaintainInfo) public maintainInfoMap;
```

Comment:
`maintainInfoMap` ==> `maintainingValidatorSetMap`

Reply:
maintainInfoMap  includes  3 status info (`not maintained` / `maintaining` / `maintained` ),  `maintainingValidatorSetMap` may not be accurate